ui/src/components/view/ResourceLimitTab.vue

```diff
@@ -119,7 +128,7 @@ export default {
 
       if (this.formLoading) return
 
-      this.form.validateFieldsAndScroll((err, values) => {
+      this.form.validateFields((err, values) => {
         if (err) {
           return
         }
```

Comment:
@utchoang I believe this should be changed?

```suggestion
    this.formRef.value.validate().then(() => {
        const values = toRaw(this.form)
```